### DISCUSSION TITLE
Python error

### DISCUSSION THREAD

**Discussion by pauldube** [Ifsttar/I-Simpa#288] [General]
I installed I-SIMPA latest version (1.3.4) and Python 3.8.10.
I opened the Tutorial 1 project and I get this error in the console:
`Error:Unable to init the Python(TM) element MD_Octave of the md_octave module
`
My Python installation seems to be OK. I cannot find any file that looks like md_octave.* in my installation folders (if that matters...)

What am I missing?

**Comment by nicolas-f** [comment]
Its a warning because the tutorial 1 project is referencing an unknown computation core "md_octave". It has no impact on the software you can ignore this error message.
best regards
